admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -87,32 +87,120 @@ You can verify that the Redis daemon is running with ``ps ax``::
  22203 ? Ssl    0:00 /usr/bin/redis-server 127.0.0.1:6379 
  
 Restart your Web server, add the appropriate entries to your ``config.php``, and 
-refresh your Nextcloud admin page. This example ``config.php`` configuration uses 
-Redis for the distributed server cache::
+refresh your Nextcloud admin page.
 
-  'memcache.distributed' => '\OC\Memcache\Redis',
-  'redis' => [
-       'host' => 'redis-host.example.com',
-       'port' => 6379,
-  ],
+Redis configuration in Nextcloud (config.php)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 For best performance, use Redis for file locking by adding this::
 
   'memcache.locking' => '\OC\Memcache\Redis',
 
+Further more, you can use Redis for the distributed server cache:: 
+
+  'memcache.distributed' => '\OC\Memcache\Redis',
+
+Additionally, you can use Redis for the local cache like so:: 
+
+  'memcache.local' => '\OC\Memcache\Redis',
```

Comment:
You can do it on single-server systems only, with more than app servers it can cause trouble. Even on single-server setups APCu should be faster. So this is possible, but not recommendable.

Reply:
I've incorporated your comments in a note.